make.sh

```diff
@@ -36,17 +36,6 @@ build_iOS() {
     ${MAKE}
 }
 
-build() {
```

Comment:
`UNAME` and `LIBARCHS` weren't being used in the `Makefile` at all. If a `MACOS_UNIVERSAL` build is wanted, wouldn't it be easier for people to run `MACOS_UNIVERSAL=yes make`?